contrib/babelfishpg_tsql/sql/sys_views.sql

```diff
@@ -843,7 +843,7 @@ left join sys.babelfish_namespace_ext ext on (nsp.nspname = ext.nspname and ext.
 left join pg_constraint const on const.conindid = I.oid and const.contype = 'u'
 -- use rownumber to get index_id scoped on each objects
 inner join 
-(select indexrelid, cast(case when indisclustered then 1 else 1+row_number() over(partition by indrelid) end as int) 
+(select indexrelid, cast(case when indisclustered then 1 else 1+row_number() over(partition by indrelid order by indexrelid) end as int) 
```

Comment:
The row_number() generated here should be same as row_number in sys.index_column
Hence added the order by clause.

Reply:
Please check the performance impact of it on SSMS query to enumerate tables.

Reply:
added query plans it